oauth2client/_helpers.py

```diff
@@ -11,12 +11,216 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """Helper functions for commonly used utilities."""
 
 import base64
+import functools
+import inspect
 import json
+import logging
+import os
+import warnings
 
 import six
+from six.moves import urllib
+
+
+__author__ = [
```

Comment:
Since the bulk of the file is someone else's code, I won't add myself.  Looking at the [history](https://github.com/google/oauth2client/commit/68a8cfe26dc8523f91ac8655aae772d8527fca2e#diff-25e202e943c248567c6fe9f00d8c5ee2), I think I'd be more comfortable removing the authors once remove `positional()`.